src/printer.ts

```diff
@@ -0,0 +1,15 @@
+import { Printer } from 'furious-commander/dist/printer'
+import { bold } from 'kleur'
+import { CommandLog, VerbosityLevel } from './command/root-command/command-log'
+
+export function createPrinter(): Printer {
+  const commandLog = new CommandLog(VerbosityLevel.Normal)
```

Comment:
Would be nice to figure out smth in a subsequent PR to share the same command log instance that the root-command has.
maybe with handling  the passed verbosity flag separately somehow here or with other solution.